frontend/src/components/Metrics/MetricsStepper/index.tsx

```diff
@@ -37,7 +39,9 @@ const MetricsStepper = () => {
         { key: isShowSourceControl, value: isSourceControlVerified },
       ]
       const activeParams = showNextButtonParams.filter(({ key }) => key)
-      hasMetrics ? setIsDisableNextButton(!activeParams.every(({ value }) => value)) : setIsDisableNextButton(true)
+      projectName && dateRange.startDate && dateRange.endDate && hasMetrics
+        ? setIsDisableNextButton(!activeParams.every(({ value }) => value))
```

Comment:
`activeParams.every(({ value }) => value)`
what does mean?
https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array/every

Reply:
First use the key value to find the displayed component, and then use the value of the object to determine whether the component was successfully validated, to determine whether next is enabled